vmdb/app/controllers/host_controller.rb

```diff
@@ -867,19 +865,19 @@ def set_credentials_verify_status(id)
       @edit[:default_verify_status] = @edit[:ws_verify_status] = @edit[:remote_verify_status]= @edit[:ipmi_verify_status] = false
     else
       host = find_by_id_filtered(Host, id.to_i)
-      if @edit[:new][:default_userid].blank? || host.ipaddress.blank?
+      if @edit[:new][:default_userid].blank?
         @edit[:default_verify_status] = false
       else
         @edit[:default_verify_status] = (@edit[:new][:default_password] == @edit[:new][:default_verify])
       end
 
-      if @edit[:new][:remote_userid].blank? || host.ipaddress.blank?
+      if @edit[:new][:remote_userid].blank?
         @edit[:remote_verify_status] = false
       else
         @edit[:remote_verify_status] = (@edit[:new][:remote_password] == @edit[:new][:remote_verify])
       end
 
-      if @edit[:new][:ws_userid].blank? || host.ipaddress.blank?
```

Comment:
Same as above (ipaddress -> hostname) for all of the above....wow, this code is all doing the same thing.